test/jasmine/tests/cone_test.js

```diff
@@ -102,7 +102,7 @@ describe('@gl Test cone autorange:', function() {
         })
         .then(function() {
             _assertAxisRanges('scaled up',
-                [-0.39, 4.39], [-0.39, 4.39], [-0.39, 4.39]
+                [-0.66, 4.66], [-0.66, 4.66], [-0.66, 4.66]
```

Comment:
Can you add one more test, to 🔒 this in a little more since it went through so many iterations: scale up the x/y/z arrays. At least one of the previous iterations had a problem with this but otherwise looked pretty good.

Reply:
there's this one:

https://github.com/plotly/plotly.js/blob/abb11e003e8c454e703668c12990e3c122500f29/test/jasmine/tests/cone_test.js#L121-L144

am I missing anything?

Reply:
It's close, but there's a difference between adding another point that *doesn't* change the minimum spacing between cones, and expanding the whole x/y/z space and therefore expanding the minimum spacing as well.

Reply:
got it. Thanks!

Reply:
done in -> https://github.com/plotly/plotly.js/pull/2641/commits/2c08357fc40005ac2505d63b374e784a0599cc9e